Tools/Scripts/webkitpy/benchmark_runner/browser_driver/linux_minibrowserwpe_driver.py

```diff
@@ -53,29 +53,31 @@ def prepare_env(self, config):
 
 
 class WPEMiniBrowserDriver(WPEMiniBrowserBaseDriver):
-    browser_name = 'minibrowser-wpe'
+    browser_name = 'minibrowser-wpe-gpu'
```

Comment:
After a second thought I think we should not rename the default minibrowser-wpe name, because the dashboard at https://browserperfdash.igalia.com/ depends on this identifier. And I guess the new dashboard also depends (not sure). So if we rename this identifier, then this is going to break the current graphs (it could not longer be possible to see the new data as continuation of the old one, it will appear like data of a new browser that starts to be tracked on the moment this lands)

So I think we should keep the default name `minibrowser-wpe` for this reason, maybe adding a comment on the code indicating that the default mini-browser uses GPU is enough.